sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -118,23 +118,35 @@ export interface SpanGraphNode {
     name: string;
 }
 
+// @public
+export enum SpanKind {
+    CLIENT = 2,
+    CONSUMER = 4,
+    INTERNAL = 0,
+    PRODUCER = 3,
+    SERVER = 1
+}
+
 // @public
 export interface SpanOptions {
     attributes?: {
         [key: string]: unknown;
     };
+    kind?: SpanKind;
+    // Warning: (ae-forgotten-export) The symbol "Link" needs to be exported by the entry point index.d.ts
+    links?: Link[];
     parent?: SpanContext | null;
 }
 
 // @public
 export class TestSpan extends NoOpSpan {
-    constructor(parentTracer: Tracer, name: string, context: OTSpanContext, kind: SpanKind, parentSpanId?: string, startTime?: TimeInput);
+    constructor(parentTracer: Tracer, name: string, context: OTSpanContext, kind: SpanKind_2, parentSpanId?: string, startTime?: TimeInput);
```

Comment:
SpanKind_2 is kind of weird, why isn't this `SpanKind`? I assume you're using the `SpanKind` from the open telemetry package for this constructor instead of the one you define in core-tracing, but that isn't intentional is it?

Reply:
Agreed. I'll remove this (I think it's unnecessary)

Reply:
Removed, it really wasn't needed.